examples/pipelines/main.tf

```diff
@@ -64,6 +64,13 @@ steps:
       "context2-name",
     ]
 
+    # hybrid runner example
+    runtime_environment {
+      name = "csdp/runner"
+      cpu = "1500"      # for 1.5 CPU
```

Comment:
This didn't quite work for us. Entering `1000` actually resulted in requesting `1000` cpus. I think its best to use a string annotation like `1500m`.